.github/workflows/pythonpublish.yml

```diff
@@ -0,0 +1,63 @@
+name: Upload Python Package
+on:
+  release:
+    types: [created]
+
+jobs:
+  deploy-linux:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.6
+      - name: Publish linux
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: |
+          docker run -v $PWD:/io quay.io/pypa/manylinux1_x86_64 "/io/build_wheels.sh"
+          pip install twine
+          twine upload wheelhouse/*.whl --skip-existing
+          twine upload dist/*.tar.gz
+  deploy-mac:
+    runs-on: macOS-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8]
```

Comment:
Is the wheel compatible with the system Python on macOS (IIRC 3.7)? They are probably not doing anything strange, so I wouldn't expect otherwise, but it would be good to sanity check.

Reply:
Not really sure how to test this, apparently system Python for macOS is available at `/usr/local/Cellar/python/` on the macOS runners according to [documentation](https://help.github.com/en/actions/language-and-framework-guides/using-python-with-github-actions#specifying-a-python-version). 

Do you suggest installing & testing the package with the python versions found there?
I.e., add:
```
/usr/local/Cellar/python/python -m pip install dist/*.whl pytest
/usr/local/Cellar/python/python -m pytest
```

Reply:
The formulation in the GitHub documentation is strange, because `/usr/local/Cellar` is a Homebrew path and thus 3rd party. The system interpreter is: `/usr/bin/python3?`

Reply:
I'm not sure what to do about the macOS python, I only find information that says macOS comes with pre installed python 2.7 and that python3 should be installed via brew or some installer from python.org, e.g. here: https://docs.python.org/3/using/mac.html

I could add a test steps in the other workflow that verifies that wheels can be installed and tests pass on that too?

Reply:
> I'm not sure what to do about the macOS python, I only find information that says macOS comes with pre installed python 2.7 and that python3 should be installed via brew or some installer from python.org, e.g. here: https://docs.python.org/3/using/mac.html

That documentation is incorrect, `/usr/bin/python3` is Python 3.7.1 on macOS. Maybe as part of the release build, you could create a virtual environment (`/usr/bin/python3 -m venv venv`), activate that environment, pip install the Python 3.7 wheel and check that it imports (and/or perhaps run the tests).

> 
> I could add a test steps in the other workflow that verifies that wheels can be installed and tests pass on that too?

Reply:
I added running the tests with both system python and the environment python. Once this is merged I'll do a test release before adding the pypi secrets to see if anything breaks